build-system/tasks/get-zindex/index.js

```diff
@@ -115,11 +122,52 @@ async function getZindexSelectors(glob, cwd = '.') {
   return filesData;
 }
 
+/**
+ * @param {string|Array<string>} glob
+ * @param {string=} cwd
+ * @return {Object}
+ */
+async function getZindexChainsInJs(glob, cwd = '.') {
```

Comment:
would this still work for references or does the value have to be on the call site? (apologies, still trying to read through the code)

Reply:
It will not follow references.

Reply:
do we have any lint rules to enforce it?

Reply:
No. I haven't encountered it so far.